src/state_machine/workload.zig

```diff
@@ -453,11 +475,18 @@ pub fn WorkloadType(comptime AccountingStateMachine: type) type {
         }
 
         fn build_create_transfers(
-            self: *Self,
+            self: *Workload,
             client_index: usize,
             transfers: []tb.Transfer,
         ) usize {
             const results = self.auditor.expect_create_transfers(client_index);
+
+            const failed_ids_count = self.build_create_transfers_failed_ids(
+                transfers,
+                results,
+            );
+            if (failed_ids_count > 0) return failed_ids_count;
```

Comment:
:thinking: I'd rather we allowed mixed failed-ids-and-regular-transfer batches. Otherwise I'm worried that every other batch will be a failed transfers batch, which would change the coverage of LSM (i.e. every other batch wouldn't be inserting objects).

Instead of removing the id (`self.transient_failures.popOrNull()`), change `transient_failures` to an arraylist (don't worry about duplicates) and select a random element (but don't remove it). Then when we get back `id_already_failed` we can verify that it is in the list.